coral-css-formgroup/src/styles/index.styl

```diff
@@ -65,12 +65,25 @@ $form-aligned-label-min-width = var(--spectrum-medium-global-dimension-size-1600
   }
 }
 
+.coral-Form-errorlabel {
+  box-sizing: border-box;
+  font-weight: var(--spectrum-medium-fieldlabel-text-font-weight);
+  font-size: var(--spectrum-medium-fieldlabel-text-size);
+  line-height: var(--spectrum-medium-fieldlabel-text-line-height);
+  -webkit-font-smoothing: subpixel-antialiased;
+  -moz-osx-font-smoothing: auto;
+  font-smoothing: subpixel-antialiased;
+  vertical-align: top;
+  display: block;
+  padding-top: var(--spectrum-medium-fieldlabel-padding-bottom);
+}
+
 /**
  * Indicates that the form is using vertical layout, where the label is placed above the field.
  */
 .coral-Form--vertical {
   .coral-Form-field {
-    margin: 0 0 $form-vertical-vmargin-max;
+    margin: 0;
```

Comment:
Are you sure ? Vertical form fields will not be spaced AFAIK. Did you check how it would look like eg in AEM Sites properties page ?

Reply:
@icaraps i think the problem here is that we should use the margin for the coral-Form-fieldwrapper instead of .coral-Form-field. will do some adjustments

Reply:
@icaraps i did the adjustment mentioned above and i think we are good both on aem and spectrum
<img width="395" alt="Screenshot 2020-05-06 at 16 41 05" src="https://user-images.githubusercontent.com/53038821/81184577-38082100-8fb9-11ea-9fe2-248424d51a26.png">
<img width="758" alt="Screenshot 2020-05-06 at 16 40 50" src="https://user-images.githubusercontent.com/53038821/81184589-3b031180-8fb9-11ea-9fd1-783a82a04b56.png">

Reply:
movin the margin from field to fieldwrapper which is the container gives us 
<img width="758" alt="Screenshot 2020-05-06 at 16 40 50" src="https://user-images.githubusercontent.com/53038821/81184727-6be34680-8fb9-11ea-9f92-e208e839931c.png">
<img width="395" alt="Screenshot 2020-05-06 at 16 41 05" src="https://user-images.githubusercontent.com/53038821/81184737-6ede3700-8fb9-11ea-9390-cd684ad5eeeb.png">
I think we are ok on both sides